test/boost/database_test.cc

```diff
@@ -575,21 +575,56 @@ SEASTAR_TEST_CASE(snapshot_list_inexistent) {
 
 
 SEASTAR_TEST_CASE(clear_snapshot) {
-    return do_with_some_data({"cf"}, [] (cql_test_env& e) {
-        take_snapshot(e).get();
-        auto& cf = e.local_db().find_column_family("ks", "cf");
+    sstring ks_name = "ks";
+    sstring table_name = "cf";
+    auto num_snapshots = 2;
 
-        unsigned count = 0;
-        lister::scan_dir((fs::path(cf.dir()) / sstables::snapshots_dir / "test"), { directory_entry_type::regular }, [&count] (fs::path parent_dir, directory_entry de) {
-            count++;
-            return make_ready_future<>();
-        }).get();
-        BOOST_REQUIRE_GT(count, 1); // expect more than the manifest alone
+    auto snapshot_name = [] (int idx) {
```

Comment:
This test checks a lot of details (like the snapshot's name) that the previous test didn't check. So I wonder if it didn't make sense to keep both tests - the small one and the larger one? But maybe not. Your call.

Reply:
Hmm, not sure either.  Maybe it'd be best to just keep the simple test since the extended test is not a pure superset of the simple test (i.e. it doesn't test exactly the same condition but does more)